tests/e2e/forms/repeat-form.wdio-spec.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+
+const constants = require('../../constants');
+const utils = require('../../utils');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+
+const xml = fs.readFileSync(`${__dirname}/../../../demo-forms/repeat-translation.xml`, 'utf8');
+const formDocument = {
+  _id: 'form:repeat-translation',
+  internalId: 'repeat-translation',
```

Comment:
The form Id you use here "repeat-translation" conflicts with the id set in the form itself:
```
repeat id="repeat"
```

I think using "repeat" as the form id is super confusing, as it's also a reserved keyword (for repeats lol). Could you please change the form XML so the form's id is actually "repeat-translation".

Reply:
Sure! Should I change anything else besides the `id="repeat"` attribute on the `<repeat>` tag?  
I don't know if there is any property depending on this attribute.

Reply:
All references to the "repeat" element have to be changed, including model paths.